System/Vector.h

```diff
@@ -507,7 +507,7 @@ namespace RTE {
 		const std::string & GetClassName() const override { return c_ClassName; }
 #pragma endregion
 
-	protected:
+	private:
```

Comment:
This doesn't affect Lua's ability to get ClassName right? Can't remember off the top of my head if luabind needs non-private to access stuff.

Reply:
Yeah, it works fine because the getter is public.